packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -632,6 +632,13 @@ class RenderEditable extends RenderBox {
     markNeedsLayout();
   }
 
+  @override
+  void systemFontsDidChange() {
+    super.systemFontsDidChange();
+    _textPainter.markNeedsLayout();
```

Comment:
I am not sure how do we test _textPainter has been marked needs layout. this also apply to all the other widget.
The current test only verify renderobject is dirty after font change.